watoi/main.m

```diff
@@ -425,7 +425,7 @@ - (NSString *) guessOurJID {
 }
 
 - (void) importMessages {
-    NSString *query = @"SELECT * FROM messages where"
+    NSString *query = @"SELECT * FROM legacy_available_messages_view where"
```

Comment:
Why is that? As I can see in the [schema](https://github.com/andreas-mausch/whatsapp-viewer/blob/master/data/msgstore.db.schema.sql), this view adds some fields (unused by us) and does some kind of filtering on `messages` table. Do you know what's going on there?

Reply:
I have no idea. I just used that view because have most of the previous fields.

Reply:
Ok, looks like this view handles "deleted" messages not being really deleted from msgstore. My only concern is that they may eventually drop this view... Will do for now.